includes/widgets/video.php

```diff
@@ -802,7 +802,20 @@ protected function render() {
 
 			$embed_options = $this->get_embed_options();
 
-			$video_html = Embed::get_embed_html( $video_url, $embed_params, $embed_options );
+			$is_static_render_mode = Plugin::$instance->frontend->is_render_mode( Frontend::RENDER_MODE_STATIC );
+			$is_edit_mode = Plugin::$instance->editor->is_edit_mode();
+			$post_id = get_queried_object_id();
+
+			if ( $is_static_render_mode || $is_edit_mode ) {
+				// Make sure that the oembed data saved in cache.
+				Embed::get_oembed_data( $video_url, $post_id );
```

Comment:
Why do you use the oEmbed in edit mode?

Reply:
I use it to cache the video thumbnail, so the next time it will be fetched from the DB and not from a remote server.
It will make the screenshot from this PR: #11751 much faster.

Reply:
@Nevoss For now remove the caching on `edit_mode` in order to avoid performance regression for existing pages.